sections/05_treat.md

```diff
@@ -78,6 +78,53 @@ able to significantly outperform a random forest classifier trained on gene
 expression changes. These approaches suggest promising directions to improve
 the efficiency of clinical trials and accelerate drug development.
 
+##### Drug repositioning
```

Comment:
This can be a `###` header, one of the major sub-sections of Treat.

I think we could also move your new sub-section to where the `Effects of drugs on transcriptomic responses` placeholder was.  The `Integrating Deep Learning with Clinical Practice` paragraph pertains to the `Categorizing patients for clinical decision making` subsection.  The other question is how this topic fits with the ligand-based screening section.  I think the order of repositioning first, screening second makes sense.  Do you agree?